### PR TITLE
CIV-10670 record judgment api tests

### PR DESCRIPTION
**Pull request by hmcts-version1-ignacio**
### API tests Record Judgment ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: e2e/api/steps.js [hunk 1]
```diff
@@ -182,7 +186,7 @@ module.exports = {
     console.log('Is PBAv3 toggle on?: ' + pbaV3);
 
     let bodyText = 'Your claim will not be issued until payment is confirmed.';
-    let headerText = '# Please now pay your claim fee\n# using the link below';
```
Comment (Azam-Hmcts) [inline e2e/api/steps.js:185]: Why we need to modify this header? I think its commited accidently.

Comment (hmcts-version1-ignacio) [reply e2e/api/steps.js:185]: Yes, true, can't figure out why this was changed. It is in Master, so I'll put it back. Thank you!

File: e2e/tests/api_tests/judgmentOnline/api_jo_record_judgment_spec_1v2_test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const judgeUser = config.judgeUserWithRegionId1;
+const legalAdvUser = config.tribunalCaseworkerWithRegionId4;
+// to use on local because the idam images are different
+// const judgeUser = config.judgeUserWithRegionId1Local;
+// const legalAdvUser = config.tribunalCaseworkerWithRegionId1Local;
+
+Feature('Record Judgment 1v2 API test spec @api-spec @api-tests-1v2 @api-jo @api-non-prod-jo');
+
+Scenario('Record Judgment Spec claim 1v2', async ({I, api_spec}) => {
+  await api_spec.createClaimWithRepresentedRespondent(config.applicantSolicitorUser, 'ONE_V_TWO_SAME_SOL');
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply e2e/tests/api_tests/judgmentOnline/api_jo_record_judgment_spec_1v2_test.js:14]: Ready.

File: e2e/tests/api_tests/judgmentOnline/api_jo_record_judgment_spec_1v1_test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const legalAdvUser = config.tribunalCaseworkerWithRegionId4;
+// to use on local because the idam images are different
+// const judgeUser = config.judgeUserWithRegionId1Local;
+// const legalAdvUser = config.tribunalCaseworkerWithRegionId1Local;
+
+
+Feature('Record Judgment 1v1 API test spec @api-spec @api-tests-1v1 @api-jo @api-non-prod-jo');
+
+Scenario('Record Judgment Spec claim 1v1', async ({I, api_spec}) => {
+  await api_spec.createClaimWithRepresentedRespondent(config.applicantSolicitorUser);
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply e2e/tests/api_tests/judgmentOnline/api_jo_record_judgment_spec_1v1_test.js:15]: Ready.

**Pull request closed without merging**